docs/docs/installation/configuration/namespace-keptn.md

```diff
@@ -5,9 +5,9 @@ comments: true
 # Namespaces and Keptn
 
 This page discusses how to restrict the namespaces
-where Keptn and the
+where the Keptn
```

Comment:
the sentence sounds like if you want to restricts where you deploy the operator which is not the case... can we rephrase it to show that it restricts what namespaces it can monitor/observe?